opcodes-dma_CUSTOM

```diff
@@ -0,0 +1,10 @@
+#
```

Comment:
No because that would be platform-dependent, while we want to keep the opcode files only ISA-extension-dependent (but I think we already discussed this). For this file it's straightforward, but I would put something like `Direct Memory Access (DMA) instructions`